vllm/config.py

```diff
@@ -1081,6 +1081,10 @@ def _parse_quant_hf_config(self):
         if quant_cfg is None:
             # compressed-tensors uses a "compression_config" key
             quant_cfg = getattr(self.hf_config, "compression_config", None)
+        if (quant_cfg is not None and quant_cfg.get("producer", {}).get(
+                "name", "").lower() == "modelopt"
+                and quant_cfg.get("quant_algo", "").lower() == "fp8"):
```

Comment:
elif would mean you wouldn't have to check if `quant_cfg` is `None` again

```suggestion
        if quant_cfg is None:
            # compressed-tensors uses a "compression_config" key
            quant_cfg = getattr(self.hf_config, "compression_config", None)
        elif (quant_cfg.get("producer", {}).get("name", "").lower() == "modelopt"
                and quant_cfg.get("quant_algo", "").lower() == "fp8"):
```